src/scripts/Game.ts

```diff
@@ -232,8 +232,17 @@ class Game {
         // Check that none of our quest are less than their initial value
         App.game.quests.questLines().filter(q => q.state() == 1).forEach(questLine => {
             const quest = questLine.curQuestObject();
-            if (quest.initial() > quest.focus()) {
-                quest.initial(quest.focus());
+            if (quest instanceof MultipleQuestsQuest) {
+                quest.quests.forEach((q) => {
+                    if (q.initial() > q.focus()) {
+                        q.initial(q.focus());
+                    }
+                });
+            } else {
```

Comment:
I think you can add a curQuestInitial fix for the multiquests with this:
```suggestion
            if (quest instanceof MultipleQuestsQuest) {
                quest.quests.forEach((q) => {
                    if (q.initial() > q.focus()) {
                        q.initial(q.focus());
                    }
                });
                questLine.curQuestInitial(quest.quests.map((q) => q.initial()));
            } else {
```

Reply:
I didn't add that originally because `curQuestInitial()` is only ever used in the QuestLine `toJSON()` function for quests that are not `MultipleQuestsQuest`. I can add it if you feel it should be there though.

Reply:
No you are right, should be fine without it :+1: